docs/getting_started/components/sip.md

```diff
@@ -0,0 +1,32 @@
+# SIP
+
+Makes a SIP connection with specified provider.
+From this connection it consumes a remote RTP stream with G711 A-law audio
+and publishes through this connection all audio tracks from engine.
+
+Supports OPUS streams only.
+
+Read more about SIP [here](https://www.rfc-editor.org/rfc/rfc3261.html).
+
+## Compatibility
+
+* [WebRTC](../peers/webrtc.md)
+* [HLS](./hls.md)
+* [RTSP](./rtsp.md)
+* [File](./file.md)
+
+## Requirements
+* The specified SIP provider needs to support G711 A-law as it will be a format in which audio from
+SIP Endpoint will be published.
+
+
+## Configuration options
+
+**Required**
+
+* `registrarCredentials` (object) - registrar credentials which allows to make a connection with a SIP provider.
```

Comment:
Should we elaborate more on the format of the credentials, or they're just in some standard format?

Reply:
Let's add a reflink to the REST API. See `s3` option in HLS for reference